indico/cli/setup.py

```diff
@@ -550,9 +569,9 @@ def _setup(self):
 
         package_root = get_root_path('indico')
         _copy(os.path.normpath(os.path.join(package_root, 'logging.yaml.sample')),
-              os.path.join(self.root_path, 'etc', 'logging.yaml'))
-        _link(os.path.join(package_root, 'htdocs'), os.path.join(self.root_path, 'web', 'htdocs'))
-        _copy(os.path.join(package_root, 'web', 'indico.wsgi'), os.path.join(self.root_path, 'web', 'indico.wsgi'),
+              os.path.join(self.config_dir_path, 'logging.yaml'))
+        _link(os.path.join(package_root, 'htdocs'), os.path.join(self.data_root_path, 'web', 'htdocs'))
```

Comment:
not needed in dev mode; since indico is installed in `--editable` mode the path to htdocs for a nginx config is pretty clear without the symlink